seq2science/rules/gene_counts.smk

```diff
@@ -96,7 +98,9 @@ elif config["quantifier"] == "salmon" and config["tpm2counts"] == "pytxi":
             cts=get_counts,
             fa=expand("{genome_dir}/{{assembly}}/{{assembly}}.fa", **config),
```

Comment:
Is it possible to add the readme as input since it is required for this rule?

Reply:
good point, difficult problem actually.

There are 2 ways to convert transcript IDs to gene names:
- using the GTF file (if it has gene names and transcript ID info)
- using mygene.info (if the README contains a taxonomy ID)

In reality, I don't think we want to use mygene.info. It would make it impossible to compare the counts table to the GTF, so downstream stuff (like ANANSE) becomes impossible. 
Of course, you still can do this. In this case you need the taxonomy ID, which is read from the genomepy README.txt

So now the next problem: what if somebody is using a custom genome? No readme, and you definitely dont want s2s to delete your data and try to download a new assembly (that might not even exist)!

Reply:
And an input function that decides if it is downloadable from genomepy, and if yes, it gives the readme.md as input, and if not some impossible input? It's perhaps better for it to crash before starting the run than at the end?

Reply:
:thinking:

Reply:
Maybe Im thinking too simple about it, anyways its not part of the PR really, so an issue would also be fine?

Reply:
Fixed. If no README.txt is available, genomepy will try to find the taxonomy online. If that fails, an error is given with a DIY solution (which should happen very rarely to begin with).